src/test/ui/consts/miri_unleashed/ptr_arith.rs

```diff
@@ -22,8 +22,6 @@ static INT_PTR_ARITH: () = unsafe {
 static PTR_ARITH: () = unsafe {
     let x = &0 as *const _;
     let _v = core::intrinsics::offset(x, 0);
-    //~^ ERROR could not evaluate static initializer
-    //~| NOTE calling intrinsic `offset`
 };
```

Comment:
I'd rather change this to something we still do not permit... what about
```rust
let _v = (x as usize) + 0;
```

Reply:
I mean isn't this just exactly what we're doing above in `INT_PTR_ARITH`?

Reply:
Oh. Yes. Well, one is a cast and one a transmute, might be worth having both? The `offset`-based test should just disappear I think, after all the entire point here is to implement those methods.

Reply:
Alternatively you can switch to testing the `arith_offset` intrinsic, which is still not const

Reply:
Done